doc/samtools-stats.1

```diff
@@ -53,6 +53,125 @@ samtools stats
 samtools stats collects statistics from BAM files and outputs in a text format.
 The output can be visualized graphically using plot-bamstats.
 
+A summary of output sections is listed below, followed by more
+detailed descriptions.
+
+.TS
+lb l .
+CHK     checksum
+SN      summary numbers
+FFQ     First fragment qualities
+LFQ     Last fragment qualities
+GCF     GC content of first fragments
+GCL     GC content of last fragments
+GCC     ACGT content per cycle
+FBC     ACGT content per cycle for first fragments only
+LBC     ACGT content per cycle for last fragments only
+BCC1    ACGT content per cycle for BC barcode
+CRC1    ACGT content per cycle for CR barcode
+OXC1    ACGT content per cycle for OX barcode
+RXC1    ACGT content per cycle for RX barcode
+QTQ1    Quality distribution for BC barcode
+CYQ1    Quality distribution for CR barcode
+BZQ1    Quality distribution for OX barcode
+QXQ1    Quality distribution for RX barcode
+IS      Insert sizes
+RL      Read lengths
+FRL     Read lengths for first fragments only
+LRL     Read lengths for last fragments only
+ID      Indel size distribution
+IC      indels per cycle
+COV     Coverage (depth) distribution
+GCD     GC-depth
+.TE
+
+Not all sections will be reported as some depend on the data being
+coordinate sorted while others are only present when specific barcode
+tags are in use.
+
+The CHK row contains distinct CRC32 checksums of read names, sequences
+and quality values.  The checksums are computed per alignment record
+and summed, meaning the checksum does not change if the input file has
+the sort-order changed.
+
+The SN section contains a series of counts, in a similar style to
```

Comment:
This is a bit vague. What do you mean by 

> a similar style to "samtools stats"

?
The users probably expect a description of what the counters and precenteges represent. This can be added later, though.